docs/tutorial.rst

```diff
@@ -85,7 +85,7 @@ Here is the full ReFrame test:
 
 .. literalinclude:: ../examples/tutorial/stream/stream_runonly.py
    :caption:
-   :pyobject: stream_test
+   :start-at: "import reframe as rfm"
```

Comment:
That was an overlook; we don't need to use `pyobject` here, but rather simply `lines`.

```suggestion
   :lines: 5-
```

Reply:
I went with lines.